lib/session.js

```diff
@@ -59,7 +59,7 @@ function storeData (input, data) {
     let val = input[i]
 
     // Delete values when users unselect checkboxes
-    if (val === '_unchecked' || val === ['_unchecked']) {
+    if ([val].flat().every((item) => item === '_unchecked')) {
```

Comment:
@BenSurgisonGDS @nataliecarey I think this was supposed to prune empty arrays `[]` from the session?

Sadly triple `=== ['_unchecked']` will never return `true` so the compiler flagged this

Reply:
Moved to https://github.com/alphagov/govuk-prototype-kit/pull/2385